genai-perf/README.md

```diff
@@ -407,13 +407,21 @@ URL of the endpoint to target for benchmarking. (default: `None`)
 
 ##### `-b <int>`
 ##### `--batch-size <int>`
```

Comment:
should we remove this to avoid confusion?
or was this left in for backwards compatibility?